### ISSUE TITLE
How do I pass the --debug argument to fvm flutter build apk?

### ISSUE THREAD

**Issue by Cir0X** [leoafarias/fvm#25]
I want to create an debug build like that:

```
fvm flutter build apk --debug
```

but it throws the following error:

```log
⚠️  Could not find an option named "debug".
Unhandled exception:
Could not find an option named "debug".
```
With npm you can pass arguments like that to the subcommand:

```
npm test -- --coverage
```

but it doesn't work with `fvm`:

```
fvm flutter build apk -- --debug 
```
Any ideas?

**Comment by Cir0X** [comment]
Should work when #26 is merged

**Comment by leoafarias** [comment]
@Cir0X For now please use the following method for calling the flutter command to pass those flags. https://github.com/leoafarias/fvm#call-local-sdk-directly

I will review your PR

**Comment by CaiJingLong** [comment]
By the way
`$ fvm flutter doctor -v` will have problems

expected:

```bash
[✓] Flutter (Channel beta, v1.10.7, on Mac OS X 10.15 19A602, locale zh-Hans-CN)
    • Flutter version 1.10.7 at /Volumes/Evo512/sdk/flutter/fvm/beta
    • Framework revision e70236e36c (3 weeks ago), 2019-10-02 09:32:30 -0700
    • Engine revision 9e6314d348
    • Dart version 2.6.0 (build 2.6.0-dev.0.0 1103600280)
```

now:

```bash
Could not find an option or flag "-v".
Unhandled exception:
Could not find an option or flag "-v".

Usage: fvm flutter [arguments]
-h, --help                       Print this usage information.
-d, --device-id                  Target device id or name (prefixes allowed).
    --[no-]version               Reports the version of this tool, on the local version.
    --[no-]suppress-analytics    Suppress analytics reporting when this command runs.
    --[no-]bug-report            Captures a bug report file to submit to the Flutter team. Contains local paths, device identifiers, and log snippets.

Run "fvm help" to see global options.
#0      Command.usageException (package:args/command_runner.dart:397:7)
#1      CommandRunner.parse (package:args/command_runner.dart:130:15)
#2      CommandRunner.run.<anonymous closure> (package:args/command_runner.dart:112:36)
#3      new Future.sync (dart:async/future.dart:224:31)
#4      CommandRunner.run (package:args/command_runner.dart:112:14)
#5      fvmRunner (package:fvm/fvm.dart:35:23)
<asynchronous suspension>
#6      main (file:///Users/cai/Documents/GitHub/fvm/bin/main.dart:4:9)
<asynchronous suspension>
#7      _startIsolate.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:303:32)
#8      _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:172:12)
```

**Comment by CaiJingLong** [comment]
Of course I can use `$ ./fvm doctor -v`.

**Comment by CaiJingLong** [comment]
@leoafarias 
I think that to solve this problem, you can solve the problem by judging whether the second parameter is flutter. If it is, then do not perform other operations, directly pass all parameters to flutter.

In the current processing mode, the `args` package may treat some flags as fvm parameters instead of flutter parameters.

**Comment by leoafarias** [comment]
@CaiJingLong that is what my PR does. When you define a sub command args package validates the flags and options or it throws an error. 

They have a parser to allow anything however it causes an error when using with sub command because the way the “help” option gets passed to the sub command.

The error is the fact that if I define a sub command that allows anything I cannot try to define an option.

Let me know if you think I am missing anything and check out the issue I opened at args and the pull request. Once they merge we should be good

**Comment by leoafarias** [comment]
This was solved with #26
